<launch>
  <env name="ROSCONSOLE_CONFIG_FILE"
       value="$(find manipulator_teleop)/config/logger.conf"/>
  <node pkg="manipulator_teleop" type="teleop" name="teleop" output="screen"/>
</launch>
